utils/spacewalk-common-channels.ini

```diff
@@ -2949,3 +2949,59 @@ gpgkey_url = https://download.opensuse.org/repositories/systemsmanagement:/Uyuni
 gpgkey_id = %(_uyuni_gpgkey_id)s
 gpgkey_fingerprint = %(_uyuni_gpgkey_fingerprint)s
 repo_url = https://download.opensuse.org/repositories/systemsmanagement:/Uyuni:/Master:/CentOS8-Uyuni-Client-Tools/CentOS_8/
+
+
+[rockylinux8]
+archs    = x86_64
+name     = RockyLinux 8 (%(arch)s)
+gpgkey_url = https://dl.rockylinux.org/pub/rocky/RPM-GPG-KEY-rockyofficial
+gpgkey_id = 6D745A60
+gpgkey_fingerprint = 7051 C470 A929 F454 CEBE 37B7 15AF 5DAC 6D74 5A60
+repo_url = https://dl.rockylinux.org/pub/rocky/8/BaseOS/%(arch)s/os/
+dist_map_release = 8
+
+[rockylinux8-appstream]
+label    = %(base_channel)s-appstream
+archs    = x86_64
+name     = rockyLinux 8 AppStream (%(arch)s)
```

Comment:
In this and the following sections the first letter of the name is written in lowercase, which is a little bit ugly.
(Btw the official way the repos are written as "Rocky Linux $releasever - <repo>")

Reply:
Good suggestion, implemented